gitalias.txt

```diff
@@ -107,6 +107,9 @@
   # branch - edit the description
   be = branch --edit-description
 
+  # list branches along with their description
+  br = !"f(){(git branch -v \"${@}\"|sed -r 's/(.) ([^ ]*)(.*)/\\2\\1 \\3/'&&git config --get-regexp ^branch.*description|sed -r '/^$/d;s/^branch\\.([^ ]*)\\.description /\\1# /')|sort -h;};f"
+
```

Comment:
I think this could replace [`branches`](https://github.com/GitAlias/gitalias/blob/a67f4445f734af44b00e97e9a493584435cb03a8/gitalias.txt#L986)

Reply:
I'll take a look at this one. IMHO you have a good base for a branch data summary. I have teammates who want to show more branch data such as the most-recent commit info e.g. who, when, etc.

Minor issue: the `sed -r` needs adjustment to make it run on vanilla macOS.

@phdru The intent of `branches` is to handle just pluralization normalization, e.g. "Whatever git branch does, do it for all the branches"..

Reply:
@joelparkerhenderson < I rewrote the `sed` without the `-r`